src/mammos_mumag/hysteresis.py

```diff
@@ -44,7 +44,9 @@ def run(
         A: Exchange stiffness constant in :math:`\mathrm{J}/\mathrm{m}`.
         K1: First magnetocrystalline anisotropy constant in
             :math:`\mathrm{J}/\mathrm{m}^3`.
-        mesh_filepath: TODO
+        mesh: The mesh can either be given as a :py:class:`~mammos_mumag.mesh.Mesh`
+        instance (for meshes available through `mammos_mumag`) or its path can be
+        specified. The only possible mesh format is `.fly`.
```

Comment:
These lines should be indented
```suggestion
            instance (for meshes available through `mammos_mumag`) or its path can be
            specified. The only possible mesh format is `.fly`.
```